config/java.header

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
The [Licenser Gradle plugin](https://github.com/CadixDev/licenser) is overall better